runtime/vm/vmprops.c

```diff
@@ -572,35 +574,45 @@ initializeSystemProperties(J9JavaVM * vm)
 		}
 	}
 
-#if JAVA_SPEC_VERSION < 12
-	/* Following system properties are defined via java.lang.VersionProps.init(systemProperties) and following settings within System.ensureProperties() */
-	{
-		const char *classVersion = NULL;
-		const char *specificationVersion = NULL;
-
-		/* Properties that always exist */
-		switch (j2seVersion) {
-		case J2SE_18:
-			classVersion = "52.0";
-			specificationVersion = "1.8";
-			break;
-		case J2SE_V11:
-		default:
-			classVersion = "55.0";
-			specificationVersion = "11";
-			break;
-		}
-		rc = addSystemProperty(vm, "java.class.version", classVersion, 0);
-		if (J9SYSPROP_ERROR_NONE != rc) {
+	if (JAVA_SPEC_VERSION == 8) {
+		specificationVersion = "1.8";
+	} else {
+		bufLen = j9str_printf(PORTLIB, NULL, 0, "%d", JAVA_SPEC_VERSION);
```

Comment:
Can you use the preprocessor for this?
https://stackoverflow.com/questions/240353/convert-a-preprocessor-token-to-a-string
https://github.com/eclipse/openj9/blob/abc3bd33fdef3079aa6e3f4a78b8dffec5b09e27/runtime/ddr/j9ddr.h#L33

Reply:
Copied the MACRO instead of including `"../ddr/j9ddr.h"`.